views/postgres/show.erb

```diff
@@ -31,6 +31,15 @@
     ["Storage", "#{@pg[:storage_size_gib]} GB"]
   ]
   
+  case @pg[:ha_type]
```

Comment:
Should we do this transformation in the serializer?

Reply:
```json
{
...
    "ha": {
        "type": 1,
        "explanation": "Active (1 standby with asyncronous replication)"
    }  
...
}
```

Reply:
Not sure... I'm thinking in the context of user facing API results... Such explanation seems weird/out of place in the API result. We can of course use separate serializer for external APIs or have an if check, but I'd like to avoid that if possible.

Reply:
We already have separate serializers for the API and Web routes.